nodes/src/nodes/tool_bland_ai/bland_driver.py

```
@@ -0,0 +1,229 @@
+        if bare_name not in self._tools:
+            raise ValueError(
+                f"Unknown tool {tool_name!r}. "
+                f"Available: {', '.join(f'{self._server_name}.{t}' for t in self._tools)}"
+            )
+
+        if not isinstance(input_obj, dict):
+            raise ValueError("Tool input must be a JSON object")
+
+        if bare_name == "make_call":
+            if not input_obj.get("phone_number"):
+                raise ValueError("phone_number is required")
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)